src/uct/ib/dc/accel/dc_mlx5.c

```diff
@@ -606,7 +590,8 @@ ucs_status_t uct_dc_mlx5_ep_fc_ctrl(uct_ep_t *tl_ep, unsigned op,
                                      txqp, txwq, MLX5_OPCODE_SEND,
                                      &av /*dummy*/, 0, op, sender_ep, 0,
                                      0, 0,
-                                     &av, uct_ib_mlx5_wqe_av_size(&av));
+                                     &av, uct_ib_mlx5_wqe_av_size(&av),
+                                     NULL); /* TODO: add GRH processing here */
```

Comment:
need to complete this to make FC work with GRH..

Reply:
it will be another PR (it will affect dc/verbs too)

Reply:
ok